tldw_Server_API/app/core/AuthNZ/create_admin.py

```
@@ -0,0 +1,160 @@
+    try:
+        users_db = await get_users_db()
+
+        # Idempotency: check if user already exists
+        try:
+            existing = await users_db.get_user_by_username(username)
+            if existing:
+                print(f"[create-admin] Admin user '{username}' already exists (id={existing.get('id', '?')}). Skipping.")
+                return True
+        except Exception:
+            # get_user_by_username may raise if schema is fresh; treat as "no user"
+            pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)